app/styles/partials/footer.scss

```diff
@@ -2,6 +2,9 @@
   display: flex;
   min-height: 100vh;
   flex-direction: column;
+  .main-container {
```

Comment:
@niranjan94 For small screen sizes, what is the expected position of rails? Because the middle portion containing information occupies all the space and placing the left and right rail along with the middle part won't make it look good.